lib/net.js

```diff
@@ -2239,7 +2248,15 @@ function onconnection(err, clientHandle) {
     clientHandle.close();
     return;
   }
-
+  if (self.blocklist && typeof clientHandle.getpeername === 'function') {
+    const remoteInfo = { __proto__: null };
+    clientHandle.getpeername(remoteInfo);
```

Comment:
It is obvious but I think we should document that it does not work if the server is behind a reverse proxy/load balancer (the common case).